sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -366,24 +471,24 @@ function buildRawSqlParameter(parameter: SqlParameter): RawSqlParameter {
     );
   }
 
-  let type: SqlParameterType;
+  let type: ParameterType;
```

Comment:
Now that this type is getting re-used for user properties, is `Parameter` still a valid term to use here?
Previously, the term `SqlParameter` was used because that was a standard term for parameters used in sql queries.

Reply:
I couldn't find any service docs w.r.t rules or sqlParameter.
I'm planning to change this to `TypeMapForRequestSerialization` and the other to `TypeMapForResponseDeserialization`